scripts/exregional_run_fcst.sh

```diff
@@ -468,6 +469,11 @@ ln_vrfy -sf ${relative_or_null} ${NEMS_YAML_FP} ${run_dir}
 
 if [ "${DO_ENSEMBLE}" = TRUE ]; then
   ln_vrfy -sf ${relative_or_null} "${FV3_NML_ENSMEM_FPS[$(( 10#${ensmem_indx}-1 ))]}" ${run_dir}/${FV3_NML_FN}
+  set_FV3nml_stoch_params cdate="$cdate" || print_err_msg_exit "\
+ Call to function to create the ensemble-based namelist for the current
+ cycle's (cdate) run directory (run_dir) failed:
+   cdate = \"${cdate}\"
+   run_dir = \"${run_dir}\""
```

Comment:
comments on Line 169 of exregional_run_fcst.sh:
`run_dir="${cycle_dir}${slash_ensmem_subdir}"
`

In jobs/JREGIONAL_RUN_FCST:
Line 72: 
`run_dir="${CYCLE_DIR}/fcst_fv3lam${SLASH_ENSMEM_SUBDIR}"`
and Line 87:
`          cycle_dir="${run_dir}" \`

So the final run_dir will be `${CYCLE_DIR}/fcst_fv3lam${SLASH_ENSMEM_SUBDIR}${slash_ensmem_subdir}`.
Need to double check here. May need to change Line 169 to just `run_dir="${cycle_dir}"`

Reply:
FYI that for the authoritative repo, in JREGIONAL_RUN_FCST:

run_dir="${CYCLE_DIR}${SLASH_ENSMEM_SUBDIR}"
cycle_dir="${CYCLE_DIR}" \

The run_dir is always one up from the cycle_dir for ensemble mode, and run_dir=cycle_dir only for deterministic runs.

Reply:
Thanks @JeffBeck-NOAA .  Because DA and other components were added into the RRFS_dev1 workflow,  we now have the following sub directories under ${CYCLE_DIR}
`anal_gsi/  fcst_fv3lam/  FV3GFS/  lbcs/  postprd/  ics/`
So it is some kind of different from the authoritative repo.

@chunhuazhou  you may need to change Line 169 of exregional_run_fcst.sh to 
`run_dir="${cycle_dir}"
`
otherwise the ${SLASH_ENSMEM_SUBDIR} will be appended twice.

Reply:
Made changes and tested the directory structures.